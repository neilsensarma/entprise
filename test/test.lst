NP_000008.1 105 ILE ASN 
NP_000008.1 108 GLY SER 
