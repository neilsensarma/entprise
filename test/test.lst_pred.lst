NP_000008.1 105 ILE ASN 0 ACADS
NP_000008.1 108 GLY SER 0.0041 ACADS
