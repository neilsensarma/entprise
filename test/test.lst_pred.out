NP_000008.1 	 105 	 ILE 	 ASN 	 0.648788393 	  	 0 	 ACADS 	 
NP_000008.1 	 108 	 GLY 	 SER 	 0.336976707 	  	 0.0041 	 ACADS 	 
